### DISCUSSION TITLE
Microsoft.Extensions.DependencyInjection version 2.0.0 is deprecated

### DISCUSSION THREAD

**Discussion by Angelina88** [fluentmigrator/fluentmigrator#1640] [General] [answered]
Hi, that package version is already deprecated and it doesn't work with higher versions. 
Could this dependency be updated? Or let project to have higher version...

**Comment by jzabroski** [comment, marked as answer]
Yes, I need to release 4.0.  Just been tied up with birth of my first child. Every weekend I tell myself i will sit down for 4 hours and wrap this up.
